djangocms_fil_bootstrap/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.0.5"
+__version__ = "0.0.6"
```

Comment:
Remove that, these should be a seperate PR just for the version bump

Reply:
Done